src/screens/ProfileScreen/elements/KarmaLock.tsx

```diff
@@ -73,11 +75,11 @@ export const KarmaLock = (props: {onPressCreatePost: () => void}) => {
               </Text>
               <Text
                 style={{
-                  fontWeight: '600',
+                  fontFamily: fonts.inter[600],
                   paddingVertical: normalize(8),
                   fontSize: normalize(14),
                   lineHeight: normalize(20),
-                  color: '#4782D7',
+                  color: COLORS.signed_primary,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `fontFamily` property has been added to the style object for the Text component, and the `color` property has been changed. This improves consistency in typography and color scheme across the application.
```diff
- fontWeight: '600',
+ fontFamily: fonts.inter[600],
- color: '#4782D7',
+ color: COLORS.signed_primary,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->